client/components/Navbar.js

```diff
@@ -1,47 +1,60 @@
 import React from 'react'
-import {connect} from 'react-redux'
-import {Link} from 'react-router-dom'
-import {logout} from '../store'
+import { useDispatch, useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+import { logout } from '../store'
 
-const Navbar = ({handleClick, isLoggedIn}) => (
-  <div>
-    <h1>FS-App-Template</h1>
-    <nav>
-      {isLoggedIn ? (
-        <div>
-          {/* The navbar will show these links after you log in */}
-          <Link to="/home">Home</Link>
-          <a href="#" onClick={handleClick}>
-            Logout
-          </a>
-        </div>
-      ) : (
-        <div>
-          {/* The navbar will show these links before you log in */}
-          <Link to="/login">Login</Link>
-          <Link to="/signup">Sign Up</Link>
-        </div>
-      )}
-    </nav>
-    <hr />
-  </div>
-)
+//Material UI Imports
+import { AppBar, Toolbar, Typography, Button, Badge, IconButton } from '@mui/material';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
 
-/**
- * CONTAINER
- */
-const mapState = state => {
-  return {
-    isLoggedIn: !!state.auth.id
-  }
-}
+const Navbar = () => {
+  const dispatch = useDispatch()
+  const { isLoggedIn, products } = useSelector(state => { return { isLoggedIn: !!state.auth.id, products: state.products } })
 
-const mapDispatch = dispatch => {
-  return {
-    handleClick() {
-      dispatch(logout())
-    }
+  const handleClick = () => {
+    dispatch(logout())
   }
+
+  return (
+    <AppBar position="static">
+      <Toolbar>
+        <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+          Fashion-JRL
+        </Typography>
+        {isLoggedIn ? (
+          <>
+            <Button component={Link} to="/home" color="inherit">Home</Button>
+            <Button component={Link} to="/" onClick={handleClick} color="inherit">Logout</Button>
+            
+            <Link to='/settings'>
+             Settings
+            </Link>  
+            {/* {
+            !!auth.avatar && <img src={`${auth.avatar}`} /> 
+            } */}
+            {
+             !!auth.isAdmin && <Link to = '/admin'> Admin </Link> 
+            }
+            <Button component={Link} to="/products" color="inherit">Products ({products.length})</Button>
+            <Link to='/cart'>Cart ({cartCount})</Link>
```

Comment:
we should have a guest cart- guests should be able to purchase/add items to cart etc